homepage/index.html

```diff
@@ -97,18 +97,19 @@ <h3>Features</h3>
 <h3>Citations</h3>
```

Comment:
I currently need to update website manually. Would you mind if we move it to github pages? We are no longer doing anything special/interactive with the website, anyways.

Reply:
https://geodynamics.github.io/burnman/

Reply:
Cool, thank you! Ok, I'll work on updates to the links in the docs / on zenodo etc etc.

Reply:
you will need to push to the newly created www branch (or make PRs if you want things to be reviewed)

Reply:
Ok, great. In that case, shall we remove the homepage directory from the repo?

Reply:
Yes, that makes sense to me.